infrastructure/setup_sigma_dao.sql

```diff
@@ -0,0 +1,45 @@
+-- #Note: Indexer should be running before executing the queries present in this file
+
+-- create a sigma_daos table if does not exit already.
+CREATE TABLE IF NOT EXISTS sigma_daos (
+  app_id bigint PRIMARY KEY, -- application id
+  app_params jsonb, -- application params with approval etc.
+  token_id bigint, -- token id
+  token_name varchar(20) -- token name
+);
+
+-- create a procedure to handle the trigger (update_app) action
+CREATE OR REPLACE FUNCTION update_app_info()
+RETURNS TRIGGER
+AS $$
+DECLARE
+    token_id bigint;
+	token_name varchar(20);
+	log_sigma text := 'U2lnbWFEQU8gY3JlYXRlZA==';
+	i record;
+BEGIN
+	FOR i IN SELECT txn FROM txn WHERE asset = NEW.asset AND typeenum=6 LOOP
+		IF i.txn::jsonb -> 'dt'->'lg' ?& ARRAY[log_sigma] THEN
+			-- store token id from db to above decalred variable token_id
+			SELECT index INTO token_id FROM public.asset ORDER BY index DESC LIMIT 1;
+			-- store token name from db to above declared variable token_name
+			SELECT (SELECT params FROM public.asset WHERE index = token_id)::jsonb -> 'an' INTO token_name;
+			-- insert values in sigma_daos table
+			INSERT INTO public.sigma_daos (
+			app_id, app_params, token_id, token_name)
+			VALUES (NEW.asset, NEW.txn, token_id, token_name);
```

Comment:
I think this is not correct
1. you are assigning `NEW.asset` as `app_id`.
2. let's remove `token_name`.

Reply:
1. We disccused about `app_id` in general
2. Ok. Done